website/.gitignore

```diff
@@ -0,0 +1,20 @@
+# Dependencies
+/node_modules
+
+# Production
+/build
```

Comment:
I don't get why we store all theses files :

![image](https://user-images.githubusercontent.com/6154987/107646611-70d8d300-6c7a-11eb-85b5-f1210ed99a60.png)

Are `website/versioned_docs` not generated by the tool ?

I think I still don't get how documentation versioning is working, for me theses versioned folders should remain in the host which serve the doc on the web.

Reply:
https://v2.docusaurus.io/docs/versioning

Those files are generated by docusaurus but not automatically, we need to call the script `docusaurus docs:version <version>`. It will then copy/paste all the docs under the documentation folder that we specify in the config file. 

We could potentially create those versioned docs with a script at build time. But each and every one of the open source project that use docusaurus, they simply generate them and leave it inside their `website` folder.